src/types/querybuilder.ts

```diff
@@ -651,6 +651,7 @@ export interface ChainableContract {
 
   orderBy: OrderBy<this>
   orderByRaw: RawQueryFn<this>
+  orderByRandom: (seed?: string) => this
```

Comment:
Since we added the method to the `ChainableContract`, the modelQuery builder needs to implement this method as well, otherwise we get typing errors

Reply:
We have discussed this privately, but I am still trying to figure out how to fix this typing issue.

I tried to duplicate the code in `DatabaseQueryBuilder` and `ModelQueryBuilder`, but I couldn't make it work properly.

Reply:
Let's discuss this on our call. In my opinion it should be super simple to implement